source/_components/lightwave.markdown

```diff
@@ -44,10 +44,16 @@ lightwave:
       name: Phone socket
     R2D4:
       name: Torch socket
+    registration:
+      name: Registration
 ```
-
+### `host`
 Where `192.168.1.2` is the ip address of your Lightwave hub.
-Each `switch` or `light` requires an `id` and a `name`. The `id` takes the form `R#D#` where `R#` is the room number and `D#` is the device number.
+### `lights` and `switches`
```

Comment:
{% linkable_title Lights and switches %}

Reply:
As above.